src/molderl_stream.erl

```diff
@@ -78,8 +75,11 @@ init(Arguments) ->
                          prod_interval = ProdInterval,
                          statsd_latency_key_in = "molderl." ++ atom_to_list(StreamName) ++ ".time_in",
                          statsd_latency_key_out = "molderl." ++ atom_to_list(StreamName) ++ ".time_out",
-                         statsd_count_key = "molderl." ++ atom_to_list(StreamName) ++ ".packets_sent"},
+                         statsd_count_key = "molderl." ++ atom_to_list(StreamName) ++ ".packets_sent",
+                         recovery_service = molderl_utils:gen_processname(recovery, StreamName)},
```

Comment:
also pass this one as argument instead of "guessing" it again here, this way we limit the ugly code in one place, easier to refactor later